scripts/base/frameworks/broker/main.zeek

```diff
@@ -86,6 +86,24 @@ export {
 	## ZEEK_BROKER_MAX_THREADS environment variable overrides this setting.
 	const max_threads = 1 &redef;
 
+    ## Max number of items we buffer at most per peer. What action to take when
+	## the buffer reaches its maximum size is determined by
+	## `peer_overflow_policy`.
+	const peer_buffer_size = 2048 &redef;
+
+    ## Configures how Broker responds to peers that cannot keep up with the
+    ## incoming message rate. Available strategies:
+	## - disconnect: drop the connection to the unresponsive peer
```

Comment:
Dominik does this then transparently trigger a `Broker::peer_lost` event, ideally with a message that indicates the cause? I think so, since the Broker manager should see the status change, but I wanted to check. This'll be a good thing to cover in cluster.log / broker.log, and it's the part where we need to see whether the cluster heals when this happens.

Reply:
This will trigger a `peer_removed` event. Listening on that event is how I've implemented a test for this: https://github.com/zeek/broker/blob/issue/gh-426/tests/btest/peering/disconnect-on-overload.cc (I plan to use that test program in a btest eventually, but it can be used manually already).

I don't think it'll have a dedicated error message, though. That's something we would need to figure out as a followup and/or use the new Broker logger integration for.